src/components/Form/Form.tsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import todos from '../../api/todos';
+import { getUserById } from '../../services/GetUserById';
+import usersFromServer from '../../api/users';
+
+type Props = {
+  onAdd: (newValue: Todo) => void;
+};
```

Comment:
```suggestion
interface Props = {
  onAdd: (newValue: Todo) => void;
};
```

for the props, the best way to describe - interface